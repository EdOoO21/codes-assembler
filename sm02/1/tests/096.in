-356 858 379 -189 892 -579 -804 674 -302 553 -906 430 815 -956 -660 -166 317 -356 -822 -478 -306 -269 -435 -731 32 654 -977 728 180 -21 114 -11 -89 -350 -502 410 719 -291 84 860 920 804 871 636 860 49 -338
