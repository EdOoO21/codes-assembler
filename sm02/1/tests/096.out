-338
49
860
636
871
804
920
860
84
-291
719
410
-502
-350
-89
-11
114
-21
180
728
-977
654
32
-731
-435
-269
-306
-478
-822
-356
317
-166
-660
-956
815
430
-906
553
-302
674
-804
-579
892
-189
379
858
-356
